patches/chrome-android-java-src-org-chromium-chrome-browser-bookmarks-BookmarkManagerOpenerImpl.java.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/chrome/android/java/src/org/chromium/chrome/browser/bookmarks/BookmarkManagerOpenerImpl.java b/chrome/android/java/src/org/chromium/chrome/browser/bookmarks/BookmarkManagerOpenerImpl.java
+index bf6d5afbc892c7c78e57c8a712f8cb4255fdb9bd..a1227424592a097a7d68c1a6911f98d474277e5e 100644
+--- a/chrome/android/java/src/org/chromium/chrome/browser/bookmarks/BookmarkManagerOpenerImpl.java
++++ b/chrome/android/java/src/org/chromium/chrome/browser/bookmarks/BookmarkManagerOpenerImpl.java
+@@ -66,7 +66,7 @@ public class BookmarkManagerOpenerImpl implements BookmarkManagerOpener {
+         Intent intent =
+                 new Intent(
+                         activity == null ? ContextUtils.getApplicationContext() : activity,
+-                        BookmarkActivity.class);
```

Comment:
what is the reason for this patch?

Reply:
Created a follow up issue https://github.com/brave/brave-browser/issues/44641